spec/models/appointment_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe Appointment, type: :model do
+  before(:each) do
+    @user = User.create(name: 'John Doe')
+    @doctor = Doctor.create(name: 'Dr. John Doe', specialization: 'Pediatrics', photo: 'https://www.google.com',
+                            user_id: @user.id)
+    @appointment = Appointment.create(datetime_of_appointment: '2024, 1, 1, 10, 00', description: 'Checkup',
+                                      user_id: @user.id, doctor_id: @doctor.id)
```

Comment:
Use some dynamically generate a time to ensure the test is agnostic of the testing date
 in this case you case use `1.years.from_now`

```ruby 
Appointment.create(
    user: seeded_user,
    doctor: seeded_doctor,
    description: "seeded description",
    datetime_of_appointment: 1.years.from_now
)
```

remember to apply this to the other cases